Ducky-Chickens/bangazon-site
# Hp rm product

# Description
Removing products from a user's inventory

## Related Ticket(s)
#11 

## Expected Behavior
When logged in on /account, proceeding to the /inventory page and selecting the ```delete``` button removes its corresponding product listing.

## Steps to Test Solution

1. `Login with a@a.com`, and navigate to `account`
1.  In Postgres, run `select * from products where user_id=1;` 
1.  On /inventory, `delete` a `product`.  The page should reflect the new inventory.
1. Check the user's products in Postgres.

## Documentation
